distributed/comm/ucx.py

```diff
@@ -59,34 +60,42 @@ def init_once():
 
     ucp.init(options=ucx_config, env_takes_precedence=True)
 
+    # Find the function, `host_array()`, to use when allocating new host arrays
+    try:
```

Comment:
I don't think we need the try except as ucx-py depends on numpy: https://github.com/rapidsai/ucx-py/blob/branch-0.14/setup.py#L73

Reply:
That may be so. However that would be like saying NumPy can rely on Pandas dependencies for code that Pandas uses from NumPy. In general we can't expect that to hold and it's a bit fragile. We should either make NumPy a requirement here or add a fallback. Seems safer to add this fallback here and it doesn't cost us much. Plus it should save us suffering later should things change in UCX-Py and we forget about this line here 🙂

Reply:
i thought i hit send last night on this comment but I didn't.  It said,

That's a good point!  Thanks